internal/distro/rhel86/package_sets.go

```diff
@@ -450,11 +450,25 @@ func edgeCommitPackageSet(t *imageType) rpmmd.PackageSet {
 			"bash-completion", "tmux", "ima-evm-utils", "audit", "podman",
 			"container-selinux", "skopeo", "criu", "slirp4netns",
 			"fuse-overlayfs", "clevis", "clevis-dracut", "clevis-luks",
-			"greenboot", "greenboot-default-health-checks",
+			"greenboot",
 		},
 		Exclude: []string{"rng-tools"},
 	}
 
+	var greenbootExtras rpmmd.PackageSet
+	if t.arch.distro.isRHEL() {
+		greenbootExtras.Include = []string{"greenboot-default-health-checks"}
+	} else {
+		greenbootExtras.Include = []string{
+			"greenboot-grub2",
+			"greenboot-reboot",
+			"greenboot-rpm-ostree-grub2",
+			"greenboot-status",
```

Comment:
we can drop this commit altogether now - centos stream 8 now has the correct packages too (and those are the same as RHEL ofc) - https://centos.mirror.server24.net/8-stream/AppStream/x86_64/os/Packages/ (also funny centos keeps the previous version for downgrades)

Reply:
I think I'd like to get this merged and then update the centos-8 repositories and manifests separately.  This PR has caused enough trouble already :)